bench/wtperf/idle_table_cycle.c

```diff
@@ -83,7 +83,7 @@ cycle_idle_tables(void *arg)
 		snprintf(uri, sizeof(uri),
 		    "%s_cycle%07d", wtperf->uris[0], cycle_count);
 		/* Don't busy cycle in this loop. */
-		__wt_sleep(1, 0);
+		(void)sleep(1);
```

Comment:
We switched to using ``__wt_sleep`` to avoid needing to write two versions of platform specific sleep functions.